README.md

```diff
@@ -196,16 +196,32 @@ All paths below are prefixed with `solution.` for fully-qualified naming and are
 
 This section contains the CORTX and third-party authentication information used to deploy CORTX on Kubernetes.
 
+A Kubernetes Secret is used to hold the various passwords and secret keys needed by the various components.
+- If the `secrets.name` field is specified, then CORTX will create and populate this Secret object, using this specified name.  For any `secrets.content` fields that are not specified or do not have a value specified, CORTX will generate a random password.
+- If the `secrets.external_secret` field is specified, then CORTX will expect a Secret object to already exist with the specified name, which contains the passwords for these fields.  This allows an admin to specify passwords outside of solution.yaml.  Note: If an `screts.external_secret` is used, then the specified Secret must define _all_ CORTX-required secrets.
```

Comment:
```suggestion
- If the `secrets.external_secret` field is specified, then CORTX will expect a Kubernetes Secret object to already exist with the specified name, which contains the passwords for these fields.  This allows an admin to specify passwords outside of solution.yaml.  Note: If a `secrets.external_secret` is used, then the specified Secret must define _all_ CORTX-required passwords.
```
There's a mix of terminology, both "passwords" and "secrets". I picked "password" here, you could go either way.

I would also be explicit about "Kubernetes Secret", to not confuse with the password-secrets.

Reply:
Agreed.  I have gone with your suggestion.